config/set/contentrepository-90.php

```diff
@@ -275,23 +279,22 @@
      * ContentContext
      */
     // ContentContext::getCurrentSite
-    // TODO: PHP
+    $methodCallToWarningComments[] = new MethodCallToWarningComment(LegacyContextStub::class, 'getCurrentSite', '!! ContentContext::getCurrentSite() is removed in Neos 9.0.');
     $rectorConfig->rule(FusionContextCurrentSiteRector::class);
-    // TODO: Fusion
     // ContentContext::getCurrentDomain
-    // TODO: PHP
-    // TODO: Fusion
+    $methodCallToWarningComments[] = new MethodCallToWarningComment(LegacyContextStub::class, 'getCurrentDomain', '!! ContentContext::getCurrentDomain() is removed in Neos 9.0.');
+    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.currentDomain', 'Line %LINE: !! node.context.currentDomain is removed in Neos 9.0.');
     // ContentContext::getCurrentSiteNode
-    // TODO: PHP
-    // TODO: Fusion
+    $methodCallToWarningComments[] = new MethodCallToWarningComment(LegacyContextStub::class, 'getCurrentSiteNode', '!! ContentContext::getCurrentSiteNode() is removed in Neos 9.0.');
```

Comment:
What can we suggest here as replacement?

Reply:
Grab subgraph and traverse up to site node, at least as hint how to do it